test/active_model_serializers/register_jsonapi_renderer_test_isolated.rb

```diff
@@ -64,13 +65,13 @@ def test_jsonapi_renderer_not_registered
           'attributes' => {
             'name' => 'Johnny Rico'
           },
-          'type' => 'users'
+        'type' => 'authors'
         }
       }
       payload = '{"data": {"attributes": {"name": "Johnny Rico"}, "type": "authors"}}'
       headers = { 'CONTENT_TYPE' => 'application/vnd.api+json' }
       post '/render_with_jsonapi_renderer', params: payload, headers: headers
-      assert expected, response.body
+      assert_equal 500, response.status
```

Comment:
I couldn't get response.body to show the exception.  It kept returning empty body.  I think this probably has something to do with it failing on render.